config/dpkg/control

```diff
@@ -1,14 +1,23 @@
-Source: python-artifacts
+Source: artifacts
 Section: python
 Priority: extra
 Maintainer: Forensic artifacts <forensicartifacts@googlegroups.com>
-Build-Depends: debhelper (>= 7.0.0), python, python-setuptools
-Standards-Version: 3.9.2
+Build-Depends: debhelper (>= 7), python-all (>= 2.7~), python-setuptools, python3-all (>= 3.2~), python3-setuptools
```

Comment:
Is there any point packaging the python with the artifacts? I imagine anyone installing this package only wants the yaml?